### PR TITLE
pack vlingo-actors.json to the nuget, ProxyGenerator bugs fixed

### LINKED ISSUE #69: Proxy generator should check for reserved names
Sometimes if interface method signatures contain parameter names like `@object`, `@event`, and all the reserved .NET names, the generated proxy won't compile.

Before generating the parameter name, the check has to be done in the "reserved" keyword list and eventually change the name of the generated parameter.

### PR DESCRIPTION
**Pull request by mihaj**
#69, #70, #71

### CODE REVIEW

File: src/Vlingo.Actors/ProxyGenerator.cs [hunk 1]
```diff
@@ -487,7 +505,12 @@ private string GetGenericConstraints(MethodInfo info)
                     .GetGenericArguments()
                     .Select(i => (i.Name, string.Join(", ", i.GetGenericParameterConstraints().Select(c => c.FullName))));
 
-                var constraints = constraintPairs.Select(p => $" where {p.Item1} : {p.Item2}");
+                if(constraintPairs.Where(p => !string.IsNullOrEmpty(p.Item2)).Count() == 0)
```
Comment (tjaskula) [inline src/Vlingo.Actors/ProxyGenerator.cs]: You can put that I, the variable and evaluate once before the 2 uses.

File: src/Vlingo.Actors/Vlingo.Actors.csproj [hunk 2]
```diff
@@ -25,10 +25,17 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="..\..\LICENSE" Pack="true" PackagePath="" />
+    <None Include="vlingo-actors.json" Pack="true" PackagePath="lib\netstandard2.0" />
```
Comment (tjaskula) [inline src/Vlingo.Actors/Vlingo.Actors.csproj]: Why we need that ?

File: src/Vlingo.Actors/vlingo-actors.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{
+  "plugin": {
```
Comment (tjaskula) [inline src/Vlingo.Actors/vlingo-actors.json]: Why this file is suddenly needed ?

File: src/Vlingo.Actors/Vlingo.Actors.csproj [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
     <!-- NuGet Metadata -->
     <IsPackable>true</IsPackable>
-    <PackageVersion>0.4.10</PackageVersion>
```
Comment (tjaskula) [inline src/Vlingo.Actors/Vlingo.Actors.csproj]: Don't bump the version. we will release this next time.

File: src/Vlingo.Actors/Vlingo.Actors.csproj (hunk 2 above)
Comment (mihaj) [reply src/Vlingo.Actors/Vlingo.Actors.csproj]: Maybe i understood wrong. Dont we need to packthe default config with the package?

File: src/Vlingo.Actors/Vlingo.Actors.csproj (hunk 4 above)
Comment (mihaj) [reply src/Vlingo.Actors/Vlingo.Actors.csproj]: Got it, no more confirmed bumps

File: src/Vlingo.Actors/Vlingo.Actors.csproj (hunk 2 above)
Comment (tjaskula) [reply src/Vlingo.Actors/Vlingo.Actors.csproj]: Sorry. my bad didn't check that you were addressing that issue as well. Good

File: src/Vlingo.Actors/vlingo-actors.json (hunk 3 above)
Comment (tjaskula) [reply src/Vlingo.Actors/vlingo-actors.json]: Ok, it's good

**Comment by mihaj** [comment]
Please check the updates

**Comment by tjaskula** [comment]
Good work @mihaj

**Pull request merged**